geoips/plugins/modules/procflows/order_based.py

```diff
@@ -0,0 +1,80 @@
+"""Processing workflow for order based data source processing."""
+
+# Python Standard Libraries
+from argparse import ArgumentParser
+
+# Third-Party Libraries
+import yaml
+
+# GeoIPS imports
+from geoips import interfaces
+from geoips.pydantic.products import ProductPluginModel
+
+
+interface = "procflows"
+family = "standard"
+name = "order_based"
+
+
+def call(
+    fnames: list[str], product_path: str, command_line_args: list[str] | None = None
```

Comment:
How is this currently called? Via the CLI?

This might not be the case yet but, eventually, `product_path` should probably be either `product` or `product_name` where it would accept the name of a registered product.